proposals/handler_test.go

```diff
@@ -384,7 +384,7 @@ func TestBallot_BallotDoubleVotedOutsideHdist(t *testing.T) {
 			require.Equal(t, b.ID(), ballot.ID())
 			return true, nil
 		})
-	th.mm.EXPECT().AddBallot(b).Return(nil)
+	th.mm.EXPECT().AddBallot(context.TODO(), b).Return(nil)
```

Comment:
`context.Background()` is meant to be used as the top-level parent context (in main, tests etc.), see: https://pkg.go.dev/context#Background

>Background returns a non-nil, empty Context. It is never canceled, has no values, and has no deadline. It is typically used by the main function, initialization, and tests, and as the top-level Context for incoming requests.
```suggestion
	th.mm.EXPECT().AddBallot(context.Background(), b).Return(nil)
```

Reply:
changed for all instances in this file